cobalt/h5vcc/h5vcc_metrics.h

```diff
@@ -39,7 +41,7 @@ class H5vccMetrics : public script::Wrappable {
   // its entirety.
   typedef MetricEventHandler H5vccMetricEventHandler;
 
-  H5vccMetrics() {}
+  H5vccMetrics() : task_runner_(base::ThreadTaskRunnerHandle::Get()) {}
```

Comment:
In the comment of this ThreadTaskRunnerHandle, it also says "// Prefer SequencedTaskRunnerHandle to this unless thread affinity is required." Have you tried to use this one?

Reply:
Unfortunately, thread-affinity matters in my case, so it needs to be Single. The underlying Chromium metrics code has checks for it all over the place using ThreadChecker. E.g., see https://source.corp.google.com/h/chrome-internal/codesearch/chrome/src/+/main:components/metrics_services_manager/metrics_services_manager.cc;l=50?q=getmetricsserviceclient.